server/src/main/java/org/apache/kafka/server/config/ReplicationConfigs.java

```diff
@@ -122,7 +122,8 @@ public class ReplicationConfigs {
     public static final String INTER_BROKER_PROTOCOL_VERSION_DEFAULT = MetadataVersion.latestProduction().version();
     public static final String INTER_BROKER_PROTOCOL_VERSION_DOC = "Specify which version of the inter-broker protocol will be used.\n" +
            ". This is typically bumped after all brokers were upgraded to a new version.\n" +
-           " Example of some valid values are: 0.8.0, 0.8.1, 0.8.1.1, 0.8.2, 0.8.2.0, 0.8.2.1, 0.9.0.0, 0.9.0.1 Check MetadataVersion for the full list.";
+           " Example of some valid values are: 0.8.0, 0.8.1, 0.8.1.1, 0.8.2, 0.8.2.0, 0.8.2.1, 0.9.0.0, 0.9.0.1 Check MetadataVersion for the full list.\n" +
+           "This configuration is only applicable in Zookeeper mode, Use `metadata.version` config for KRaft instead.";
```

Comment:
pardon me, do we have "metadata.version" config? According to `kafka-storage.sh`, the related config name is `release-version`

Reply:
Hello @chia7712, When I read the [KIP-900](https://cwiki.apache.org/confluence/display/KAFKA/KIP-900%3A+KRaft+kafka-storage.sh+API+additions+to+support+SCRAM+for+Kafka+Brokers), It seens that we should use `kafka-storage [--release-version RELEASE_VERSION]`, but in [KIP-778](https://cwiki.apache.org/confluence/display/KAFKA/KIP-778%3A+KRaft+to+KRaft+Upgrades), if we want upgrade, should use `kafka-features --bootstrap-server [--metadata METADATA]` and `kafka-storage [--metadata-version VERSION] `